src/test/receiveMessageTest.test.js

```diff
@@ -1,32 +1,33 @@
-const { Client, GatewayIntentBits, Events } = require('discord.js');
-require('dotenv/config'); // Load environment variables
+const { Client, GatewayIntentBits, Events } = require("discord.js");
+require("dotenv/config");
 
 const client = new Client({
   intents: [
     GatewayIntentBits.DirectMessages, // Allows the bot to receive direct messages
     GatewayIntentBits.MessageContent, // Required to read message content
```

Comment:
```suggestion
    GatewayIntentBits.DirectMessages,
    GatewayIntentBits.MessageContent,
```
pretty self-explanatory here too..